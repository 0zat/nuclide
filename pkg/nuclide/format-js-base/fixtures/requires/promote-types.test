'use strict';

import type one from 'one';

var two: ?one = new one();
