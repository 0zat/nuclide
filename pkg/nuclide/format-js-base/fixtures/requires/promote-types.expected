'use strict';

var one = require('one');

var two: ?one = new one();
